app/views/inputBox.view.js

```diff
@@ -1,7 +1,9 @@
 var MainView = Backbone.View.extend({
     el: $("#my-app"),
     initialize: function (options) {
+
         this.collection = new RectanglesCollection([new Rectangle(), new Rectangle()]);
+        this.collection.once('update', this.drawRectangles.bind(this));
         this.collection.on('change', this.drawRectangles.bind(this));

```

Comment:
when i load page i fetch collection from database and it counts as an update, though if i give some new parameters through input it only counts as a change. should i combine those methods into one line or leave it like this because i need update event only once?